kv/memdb_buffer.go

```diff
@@ -25,6 +25,13 @@ import (
 	"github.com/pingcap/tidb/terror"
 )
 
+// Those limits is enforced to make sure the transaction can be well handled by TiKV.
+var (
+	EntrySizeLimit  = 8 * 1024 * 1024
```

Comment:
For simplicity, this is not set as session variables, so I set it as the same as real TiKV, which is 8MB.